DNN Platform/Library/Common/Utilities/Config.cs

```diff
@@ -632,6 +640,61 @@ public static string Save(XmlDocument xmlDoc, string filename)
             return retMsg;
         }
 
+        public static string SaveNonConfig(string document, string filename)
```

Comment:
I don't like this being a public API, could we make this internal and use internals visible to?  or something

Reply:
Can we just move this into `ConfigConsoleControllers`?

Reply:
@bdukes Yeah, we can move it.  I was just following the existing pattern.  :)  
@mitchelsellers I thought you'd say that.  :D

Reply:
Done.  :)